src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2603,6 +2605,9 @@ limitations under the License.
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
+            <include cluster="Ethernet Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="WiFi Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
```

Comment:
The fact that this cluster name in XML does not match the spec ("WiFi" vs "Wi-Fi") is pretty unfortunate?  Are you able to do a followup to fix that?

If not, let me know, please.

Reply:
Sure, I will raise a PR to try to fix this.